ors-engine/src/main/java/org/heigit/ors/snapping/SnappingRequest.java

```diff
@@ -6,17 +6,22 @@
 import com.graphhopper.routing.weighting.Weighting;
 import com.graphhopper.storage.GraphHopperStorage;
 import com.graphhopper.util.PMap;
+import lombok.Getter;
+import lombok.Setter;
 import org.heigit.ors.common.ServiceRequest;
 import org.heigit.ors.matrix.MatrixSearchContext;
 import org.heigit.ors.matrix.MatrixSearchContextBuilder;
+import org.heigit.ors.routing.RoutingProfile;
 import org.heigit.ors.routing.RoutingProfileType;
 import org.heigit.ors.routing.WeightingMethod;
 import org.heigit.ors.routing.graphhopper.extensions.ORSWeightingFactory;
 import org.heigit.ors.util.ProfileTools;
-import org.heigit.ors.routing.RoutingProfile;
 import org.locationtech.jts.geom.Coordinate;
 
 public class SnappingRequest extends ServiceRequest {
+    @Getter
+    @Setter
+    private String profileName;
```

Comment:
1. Good point. There is even more potential. All properties are accessed from the outside and have boilerplate getter at the moment. Let's add a global @Getter. A setter is only needed for profileName and maximumLocations since the other properties are set in the constructor.
2. Please elaborate on this, I don't understand.

Reply:
https://github.com/GIScience/openrouteservice/pull/1908